CONTRIBUTING.md

```diff
@@ -30,9 +29,9 @@ When submitting a PR make sure that it:
 
 1. Changes are automatically linted and tested using the [`ct` tool](https://github.com/helm/chart-testing) as a [GitHub action](https://github.com/helm/chart-testing-action). Those tests are based on `helm install`, `helm lint` and `helm test` commands and provide quick feedback about the changes in the PR. For those tests, the chart is installed on top of [kind](https://github.com/kubernetes-sigs/kind) and this step is not blocking (as opposed to 3rd step).
 2. Changes are manually reviewed by Bitnami team members.
-3. Once the changes are accepted, the PR is tested into the Bitnami CI pipeline, the chart is installed and tested (verification and functional tests) on top of different k8s platforms.
-4. If everything works fine, the `bitnami-bot` will add a new commit to the PR updating all the Docker images tags and dependencies. If something fails, the review process continues.
-5. When the PR passes all tests it is merged in the GitHub master branch and the new version of the chart is pushed to the registry. Please note that may be a slight difference between the appearance of the code in GitHub and the chart in the registry.
+3. Once the changes are accepted, the PR is tested (if needed) into the Bitnami CI pipeline, the chart is installed and tested (verification and functional tests) on top of different k8s platforms.
+5. When the PR passes all tests it is merged in the GitHub `master` branch.
```

Comment:
Shouldn't this be `4.` instead of `5.`?

Also, should we make explicit that the merge is not performed automatically and needs from manual intervention?